### DISCUSSION TITLE
【问题】react 生态中是否也有类似 unplugin-vue-components 的库呢？

### DISCUSSION THREAD

**Discussion by wangrongding** [unplugin/unplugin-vue-components#867] [Q&A]
您好~ 我正在寻找 react 生态中类似 unplugin-vue-components 库，但是找了一圈，好像没有。请问目前存在这种吗？或者说未来这个仓库会不会新起一个分支为 react 生态做这方面的事情呢？

Hello~ I'm looking for a library like unplugin-vue-components in the react ecosystem, but after looking around, there doesn't seem to be one. Does this currently exist? Or will there be a new branch of this repository to do this for the react ecosystem in the future?

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by enpitsuLin** [comment]
react依靠完善的ts功能应该不太需要这种库吧而且react也没有全局组件的概念，vue是因为模板和ts的需求才有这个库的，所以如果你使用tsx写vue也是不太需要这个库的。

react直接ComponentName+emmet展开自动引入还能获得第一时间的类型检查，如果加上一步解析引入到global类型下属于更浪费时间了

**Comment by wangrongding** [comment]
@enpitsuLin  感谢回复，主要是也想在 react 中使用组件时，可以省掉 import 这个环节，和 vue 一样的这种自动引入确实方便不少，目前还没看到有什么好的方案。

**Comment by lishaobos** [comment]
> @enpitsuLin 感谢回复，主要是也想在 react 中使用组件时，可以省掉 import 这个环节，和 vue 一样的这种自动引入确实方便不少，目前还没看到有什么好的方案。

unplugin-auto-import 使用没
